build/Targets/Settings.props

```diff
@@ -145,7 +141,12 @@
              AssemblyFile="$(NuGetPackageRoot)\Roslyn.Build.Util\$(RoslynBuildUtilVersion)\lib\net46\Roslyn.MSBuild.Util.dll"
              Condition="'$(OS)' == 'Windows_NT'" />
 
-  <Import Project="$(RoslynDiagnosticsPropsFilePath)" Condition="'$(UseRoslynAnalyzers)' == 'true' AND Exists('$(RoslynDiagnosticsPropsFilePath)')" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeQuality.Analyzers" Version="$(MicrosoftCodeQualityAnalyzersVersion)" PrivateAssets="all" />
```

Comment:
`Microsoft.CodeAnalysis.FxCopAnalyzers` does not include `Roslyn.Diagnostics.Analyzers`, which was the reason we had created `Microsoft.Net.RoslynDiagnostics`, in case the requirements diverged significantly. I am fine with either referencing individual packages or referencing `Microsoft.Net.RoslynDiagnostics` as before.

Reply:
Sorry, I should have been more clear. We should reference `Microsoft.CodeAnalysis.FxCopAnalyzers` and `Roslyn.Diagnostics.Analyzers`, and not reference the 3 parts of FxCopAnalyzers separately.

Reply:
@333fred This is not possible. Omitting the individual references results in losing the `PrivateAssets="all"` attribute.